test/fixtures/box/label-dynamic.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
                 content: 'This is dynamic!',
               },
               enter({element}) {
-                element.options.label.display = true;
+                element.elements[0].options.display = true;
```

Comment:
@kurkle as reported https://github.com/chartjs/chartjs-plugin-annotation/pull/698#issuecomment-1054016151, to change the label options at runtime it's a little bit more complex.

Reply:
We could help that use case by providing a `label` getter in box annotation
```js
get label() { return this.elements && this.elements[0]; }
```